src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHandlerManager.cs

```diff
@@ -29,6 +29,13 @@ public LanguageServiceHandlerManager(UnconfiguredProject project, ICommandLinePa
             _commandLineParser = commandLineParser;
             _handlerProvider = handlerProvider;
             _logger = logger;
+            CommandLineNotifications = new OrderPrecedenceImportCollection<Action<string, BuildOptions, BuildOptions>>(projectCapabilityCheckProvider: project);
```

Comment:
Wait, how do you have this both `[ImportMany]` on the property, *and* explicitly initialized in the constructor?